.gitattributes

```diff
@@ -0,0 +1,304 @@
+*.fish      text eol=lf
+*.inc       text
+*.ipynb     text
+*.h         text
+*.ksh       text eol=lf
+*.ps1       text
+*.pxd       text diff=python
+*.py        text diff=python
+*.py3       text diff=python
+*.pyi       text diff=python
+*.pyw       text diff=python
+*.pyx       text diff=python
+*.qss       text
+*.r         text
+*.R         text
+*.rb        text
+*.rmd       text
+*.Rmd       text
+*.rnw       text
+*.Rnw       text
+*.sh        text
+*.zsh       text eol=lf
+
+# Documentation
+*.adoc      text
+*.latex     text
+*.LaTeX     text
+*.markdown  text
+*.md        text
+*.po        text
+*.pot       text
+*.rd        text
+*.Rd        text
+*.rst       text
+*.tex       text
+*.TeX       text
+*.tmpl      text
+*.tpl       text
+
+# Web
+*.atom      text
+*.css       text
+*.htm       text
+*.html      text
+*.js        text
+*.jsx       text
+*.json      text
+*.php       text
+*.pl        text
+*.rss       text
+*.sass      text
+*.scss      text
+*.xht       text
+*.xhtml     text
+
+# Configuration
+*.cfg       text
+*.cnf       text
+*.conf      text
+*.config    text
+*.desktop   text
+*.inf       text
+*.ini       text
+*.plist     text
+*.toml      text
+*.xml       text
+*.yml       text
+*.yaml      text
+
+# Plain text data
+*.cdl       text
+*.csv       text
+*.dif       text
+*.geojson   text
+*.gml       text
+*.kml       text
+*.sql       text
+*.tab       text
+*.tsv       text
+*.wkt       text
+
+# Other text files
+*.diff      -text
+*.patch     -text
+
+# Special files
+.*rc            text
+.checkignore    text
+.ciocheck       text
+.ciocopyright   text
+.editorconfig   text
+.gitattributes  export-ignore
+.gitconfig      export-ignore
+.gitignore      export-ignore
+.gitmodules     export-ignore
+.gitkeep        export-ignore
+*.lektorproject text
+.nojekyll       text
+.project        text
+
+AUTHORS         text
+CHANGELOG       text
+CHANGES         text
+CONTRIBUTING    text
+INSTALL         text
+license         text
+LICENSE         text
+NEWS            text
+NOTICES         text
+readme          text
+*README*        text
+RELEASE         text
+TODO            text
+
+browserslist    text
+contents.lr     text
+makefile        text
+Makefile        text
+MANIFEST.in     text
+
+
+# Declare files that will always have CRLF line endings on checkout.
+*.bat           text eol=crlf
+*.cmd           text eol=crlf
+*.vbs           text eol=crlf
+*.vb            text eol=crlf
+
+
+# Denote all files that are truly binary and should not be modified.
+
+# Executable
+*.app       binary
+*.bin       binary
+*.deb       binary
+*.dll       binary
+*.dylib     binary
+*.elf       binary
+*.exe       binary
+*.ko        binary
+*.lib       binary
+*.msi       binary
+*.o         binary
+*.obj       binary
+*.pyc       binary
+*.pyd       binary
+*.pyo       binary
+*.rdb       binary
+*.Rdb       binary
+*.rdx       binary
+*.Rdx       binary
+*.rpm       binary
+*.so        binary
+*.sys       binary
```

Comment:
All of these extentions are necessary?

Reply:
See [my comment](https://github.com/spyder-ide/qtpy/pull/278#issuecomment-963716528)